tutorials/conversion_onnx/keras2onnx.ipynb

```diff
@@ -0,0 +1,232 @@
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "INFO:tensorflow:Assets written to: ../models/resnet_savedmodel/assets\n"
+     ]
+    }
+   ],
+   "source": [
+    "savedmodel_dir = '../models/resnet_savedmodel'\n",
+    "tf.saved_model.save(model, savedmodel_dir)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Convert to ONNX."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/home/yangliu/miniconda3/lib/python3.8/runpy.py:127: RuntimeWarning: 'tf2onnx.convert' found in sys.modules after import of package 'tf2onnx', but prior to execution of 'tf2onnx.convert'; this may result in unpredictable behaviour\n",
+      "  warn(RuntimeWarning(msg))\n",
+      "2022-02-01 14:10:56,018 - INFO - Signatures found in model: [serving_default].\n",
+      "2022-02-01 14:10:56,019 - INFO - Output names: ['predictions']\n",
+      "WARNING:tensorflow:From /home/yangliu/miniconda3/lib/python3.8/site-packages/tf2onnx/tf_loader.py:706: extract_sub_graph (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.\n",
+      "Instructions for updating:\n",
+      "Use `tf.compat.v1.graph_util.extract_sub_graph`\n",
+      "2022-02-01 14:10:58,785 - WARNING - From /home/yangliu/miniconda3/lib/python3.8/site-packages/tf2onnx/tf_loader.py:706: extract_sub_graph (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.\n",
+      "Instructions for updating:\n",
+      "Use `tf.compat.v1.graph_util.extract_sub_graph`\n",
+      "2022-02-01 14:10:59,963 - INFO - Using tensorflow=2.4.1, onnx=1.9.0, tf2onnx=1.9.3/1190aa\n",
+      "2022-02-01 14:10:59,963 - INFO - Using opset <onnx, 9>\n",
+      "2022-02-01 14:11:03,452 - INFO - Computed 0 values for constant folding\n",
+      "2022-02-01 14:11:05,876 - INFO - Optimizing ONNX model\n",
+      "2022-02-01 14:11:11,212 - INFO - After optimization: Add -1 (18->17), BatchNormalization -53 (53->0), Const -162 (270->108), GlobalAveragePool +1 (0->1), Identity -57 (57->0), ReduceMean -1 (1->0), Squeeze +1 (0->1), Transpose -213 (214->1)\n",
+      "2022-02-01 14:11:11,764 - INFO - \n",
+      "2022-02-01 14:11:11,764 - INFO - Successfully converted TensorFlow model ../models/resnet_savedmodel to ONNX\n",
+      "2022-02-01 14:11:11,765 - INFO - Model inputs: ['input_1']\n",
+      "2022-02-01 14:11:11,765 - INFO - Model outputs: ['predictions']\n",
+      "2022-02-01 14:11:11,765 - INFO - ONNX model is saved at ../models/resnet_savedmodel.onnx\n"
+     ]
+    }
+   ],
+   "source": [
+    "onnx_savedmodel = '../models/resnet_savedmodel.onnx'\n",
```

Comment:
This is the code which requires the model. Is it possible to skip the test only for this line in the notebook?